test/NuGet.Clients.Tests/NuGet.PackageManagement.VisualStudio.Test/CpsPackageReferenceProjectTests.cs

```diff
@@ -738,7 +738,7 @@ public async Task TestPackageManager_UpgradePackageForAllProjects_Success()
             }
         }
 
-        [Fact]
+        [Fact(Skip = "https://github.com/NuGet/Home/issues/10208")]
         public async Task TestPackageManager_UpgradePackageFor_TopParentProject_Success()
```

Comment:
If this one have problem then there're many very similar unit tests should have same problem.
isn't it just 1-2 time thing due to other factors?